README.md

```diff
@@ -143,40 +147,25 @@ For other platforms, please use the [Vagrantfile](#setting-up-vagrant) that we h
 
 ## Dependencies
 
-### Debian, Ubuntu and related distributions
-
-```console
-$ sudo apt-get install    \
-      pkg-config          \
-      libsystemd-dev      \
-      libdbus-glib-1-dev  \
-      build-essential     \
-      libelf-dev          \
-      libseccomp-dev      \
-      libclang-dev        \
-      libssl-dev
+Install `just`:
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf <https://just.systems/install.sh> | bash -s
```

Comment:
Link to just's github repo instead of giving specific instructions, so users can choose whatever option suites them best.